torchtext/experimental/functional.py

```diff
@@ -9,6 +9,13 @@ def func(tok_iter):
     return func
 
 
+def vector_func(vector):
```

Comment:
Is this for the experimental vector? Why not to write this as `nn.Module`?

Reply:
This is because this will only be used with the legacy pipeline. I believe you did something similar with `vocab_func`

Reply:
let's not add this for now because it's related to the legacy code. If you need a function like this, just add it to `examples/data_pipelines/transform.py`.